src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -890,6 +892,40 @@ class LeVADocumentUseCase extends DocGenUseCase {
         return this.createDocument(documentType, repo, data_, [:], modifier, null, watermarkText)
     }
 
+    String createTRC(Map repo = null, Map data = null) {
+        def documentType = DocumentType.TRC as String
+
+        def sections = this.jiraUseCase.getDocumentChapterData(documentType)
```

Comment:
@martsec the watermark is not contained anymore. Please check out the other create methods and readd.

Reply:
@martsec I'll add myself in a fix branch and close